src/components/SuggestionsList/SuggestionsListItem.tsx

```diff
@@ -18,7 +18,7 @@ export interface SuggestionItem {
 }
 
 export interface SuggestionsListItemProps extends SuggestionItem, NestableBaseComponentProps {
-  searchText: string;
+  searchText?: string;
```

Comment:
@richardtagger Seems I'll need to make some of the features optional, like I won't need to highlight search text in order to use SuggestionsListItem in place of the IPersona based list items...

Reply:
why cant you just pass it down?

Reply:
the current search value I mean